src/Chirp.Web/Pages/Claims.cshtml.cs

```diff
@@ -0,0 +1,40 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Chirp.Web.Pages;
+
+public class ClaimsModel : PageModel
+{
+    public string? NameID { get; set; }
+    public string? Username { get; set; }
+    public string? Name { get; set; }
```

Comment:
Might have to remove this since Name can be null depending if the Github user states their name in their settings or not.